core/src/main/scala/kafka/tools/StreamsResetter.java

```diff
@@ -163,6 +193,15 @@ private void parseArguments(final String[] args) throws IOException {
             .ofType(String.class)
             .withValuesSeparatedBy(',')
             .describedAs("list");
+        consumerPropertyOption = optionParser.accepts("consumer-property", "A mechanism to pass user-defined properties in the form key=value to the consumer")
```

Comment:
We are using `AdminClient`, `ZkUtils`, and a `KafkaConsumer` -- do we need to allow properties for all three?